Add docker

alekseyshiyanov opened the pull request.

alekseyshiyanov committed:
Добавлены файлы

alekseyshiyanov committed:
Добавлены файлы

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Добавлены файлы

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Перемещены без изменений

alekseyshiyanov committed:
Перемещены без изменений

alekseyshiyanov committed:
Перемещены с изменениями

alekseyshiyanov committed:
Удалено

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Перемещены без изменений

alekseyshiyanov committed:
Добавлены тесты для сервера

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Удалены файлы

alekseyshiyanov committed:
Перемещены файлы

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Внесены изменения

alekseyshiyanov committed:
Update BookingRepositoryTest.java

Umka7389 commented on docker-compose.yml:
```diff
@@ -0,0 +1,48 @@
+    depends_on:
+      - db
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://db:5432/shareit
```
spring.datasource.username и password  было бы тоже неплохо вынести в переменные, тк в зависимости от того на каком стенде будет деплоиться приложение их значение может быть разным

Umka7389 reviewed (commented):
Молодец, отличная работа! Рекомендация в комментарии

alekseyshiyanov committed:
Update docker-compose.yml

Внесены изменения в соответствии с рекомендациями.
Изменены переменные окружения для контейнера базы данных

alekseyshiyanov committed:
Update 01-schema.sql

Внесены изменения

alekseyshiyanov committed:
Update application.properties

Внесены изменения в соответствии с рекомендациями

Umka7389 reviewed (commented):
Молодец, отличная работа!
